src/status_im2/contexts/activity_center/notification/reply/style.cljs

```diff
@@ -0,0 +1,8 @@
+(ns status-im2.contexts.activity-center.notification.reply.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def tag
+  {:background-color colors/white-opa-10})
+
+(def tag-text
+  {:color colors/white})
```

Comment:
While we can't configure the zprint formatter to add a newline character at the end of files, it'd be nice if all collaborators configured their editors to add a new line automatically. Hopefully we can automate this soon.

This is a strict recommendation from the Clojure Style Guide. https://guide.clojure.style/#terminate-files-with-a-newline